ipatests/test_integration/test_krbtpolicy.py

```diff
@@ -177,3 +177,54 @@ def test_krbtpolicy_otp(self):
             reset_to_default_policy(master, USER1)
             self.master.run_command(['rm', '-f', armor])
             master.run_command(['ipa', 'config-mod', '--user-auth-type='])
+
+    def test_krbtpolicy_jitter(self):
+        """Test jitter lifetime with no authentication indicators """
+        master = self.master
+        tasks.kinit_admin(self.master)
+
+        master.run_command(['kinit', USER1], stdin_text=PASSWORD + '\n')
+        result = master.run_command('klist | grep krbtgt')
+
+        assert maxlife_within_policy(result.stdout_text, MAXLIFE,
+                                     slush=60) is True
+
+        tasks.kdestroy_all(master)
+
+    def test_krbtpolicy_jitter_auth(self):
+        """Test jitter lifetime with various authentication indicators """
+        master = self.master
+        tasks.kinit_admin(self.master)
+
+        master.run_command(['ipa', 'user-mod', USER1,
+                            '--user-auth-type', 'otp'])
+        master.run_command(['ipa', 'config-mod',
+                            '--user-auth-type', 'otp'])
+
+        master.run_command(['kinit', USER1], stdin_text=PASSWORD + '\n')
+        result = master.run_command('klist | grep krbtgt')
+
+        assert maxlife_within_policy(result.stdout_text, MAXLIFE,
+                                     slush=60) is True
+
+        reset_to_default_policy(master, USER1)
+        master.run_command(['ipa', 'config-mod', '--user-auth-type='])
+
+        """ Test jitter lifetime for pkinit """
```

Comment:
Probably should be a normal comment (or separate function, depending)?

Reply:
Moved to its own function, but it's very redundant since it's exactly the same as the otp function. Should I only have one function, or should I keep both?